### PR TITLE
incorporate popup component

### PR DESCRIPTION
**Pull request by jon-stevens**
This component was originally using a local popup component. This PR incorporates the global-popup component instead.

### CODE REVIEW

File: toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss [hunk 1]
```diff
@@ -105,6 +105,64 @@
...
+
+		.js & {
+			position: absolute;
+			width: 90vw;
+			border-top: 3px solid #007e99;
+			border: 2px solid #00b8b0;
+			box-shadow: 0 4px 6px 2px rgba(0, 0, 0, 0.2);
+			border-radius: 3px;
+			padding: 30px;
+			margin: -15px 30px 30px 0;
```
Comment (benjclark) [inline toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss:127]: should these be using the spacing function? e.g. `spacing(30)`

apologies if i've asked this before on another PR and forgotten

File: toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss [hunk 2]
```diff
@@ -105,6 +105,64 @@
 		font-weight: bold;
 	}
 
+	&__author-modal {
```
Comment (benjclark) [inline toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss:108]: Are there any values in here that are repeated and therefore should live in 10-settings?

File: toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss (hunk 1 above)
Comment (jon-stevens) [reply toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss:127]: No idea! Never heard of it until now 😆

Comment (benjclark) [reply toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss:127]: haha, it's a function in default brand context the purpose of which is to constrain developers to only use spacing values from a defined map per brand - for sake of consistency

File: toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss (hunk 2 above)
Comment (jon-stevens) [reply toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss:108]: Yep, will do

**Pull request merged**